tests/encodehelp.h

```diff
@@ -25,7 +25,7 @@
 #include "interface/VideoEncoderDefs.h"
 #include <getopt.h>
 
-static const int kIPeriod = 30;
```

Comment:
It just temp code here, just as my commit log. Actually I've revert it on my local site since P frame can work now.